### PR TITLE
Feature/random movies button

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { observerListener } from '../../shared/observer';
 
 
 
-export const Button = ({ onClick, id, text, type, className }) => {
+export const Button = ( onClick, id, text, type, className ) => {
```
Comment (Veryyapeee) [inline src/components/Button/Button.js]: Why did you changed the way that arguments are passed here?

Comment (T0ffik) [reply src/components/Button/Button.js]: Ah I got some error so I tried fixed this somehow, I gonna change it to original version

**Review by maciejopalinski — APPROVED** [review]
Except what @Veryyapeee requested, looks good to me!

File: src/components/Button/Button.js (hunk 1 above)
Comment (T0ffik) [reply src/components/Button/Button.js]: Done already

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**